api/v1alpha1/sandbox_types.go

```diff
@@ -116,6 +124,10 @@ type SandboxStatus struct {
 	// SandboxIp is the ip address allocated to the sandbox.
 	// +optional
 	SandboxIp string `json:"sandboxIp,omitempty"`
+
+	// LatestRevision, if not empty, indicates the latest revision of the Sandbox.
+	// +optional
+	LatestRevision string `json:"latestRevision,omitempty"`
```

Comment:
what latest means here? is it the actual revision of the pod, or the desired revision of the sandbox?